packages/shared/src/components/cards/PostCard.tsx

```diff
@@ -67,89 +65,108 @@ export const PostCard = forwardRef(function PostCard(
       <CardTextContainer>
-        <CardHeader>
-          <SourceButton post={post} style={{ marginRight: '0.875rem' }} />
-          {featuredCommentsToButtons(post.featuredComments, setSelectedComment)}
-          <OptionsButton
-            onClick={(event) => onMenuClick?.(event, post)}
-            post={post}
+        {isV1 && (
+          <PostCardHeader
+            source={post.source}
+            postLink={post.permalink}
+            onMenuClick={(event) => onMenuClick?.(event, post)}
           />
-        </CardHeader>
+        )}
         <CardTitle className={classNames(className, postHeadingFont)}>
           {post.title}
         </CardTitle>
       </CardTextContainer>
-      <CardSpace />
-      <PostMetadata
-        createdAt={post.createdAt}
-        readTime={post.readTime}
-        className="mx-4"
-      />
-      {!showImage && (
-        <PostAuthor
-          post={post}
-          selectedComment={selectedComment}
-          className="mx-4 mt-2"
+      <Containter className="mb-8 tablet:mb-0">
+        <CardSpace />
+        <PostMetadata
+          createdAt={post.createdAt}
+          readTime={post.readTime}
+          className="mx-4"
         />
-      )}
-      {showImage && (
-        <CardImage
-          imgAlt="Post Cover image"
-          imgSrc={post.image}
-          fallbackSrc="https://res.cloudinary.com/daily-now/image/upload/f_auto/v1/placeholders/1"
-          className="my-2"
-        >
-          {post.author && (
-            <div
-              className={classNames(
-                'absolute flex items-center py-2 px-3 text-theme-label-secondary bg-theme-bg-primary z-1 font-bold typo-callout w-full',
-                selectedComment ? 'invisible' : styles.authorBox,
-              )}
-            >
-              <ProfilePicture size="small" user={post.author} />
-              <span className="flex-1 mx-3 truncate">{post.author.name}</span>
-              <FeatherIcon className="text-2xl text-theme-status-help" />
-            </div>
+      </Containter>
+      <Containter>
+        {postCardVersion === 'v2' && (
+          <PostFooterOverlay
+            className={classNames(
+              'rounded-b-12',
+              insaneMode
+                ? 'relative tablet:absolute tablet:right-0 tablet:bottom-0 tablet:left-0 mt-2 tablet:mt-0 tablet:border-0 border-t border-theme-divider-tertiary'
+                : 'absolute right-0 bottom-0 left-0',
+            )}
+            postLink={post.permalink}
+            source={post.source}
+            author={post.author}
+            insaneMode={insaneMode}
+          />
+        )}
+        {!showImage && (
+          <PostAuthor post={post} className="hidden tablet:flex mx-4 mt-2" />
+        )}
+        {showImage && (
+          <CardImage
+            imgAlt="Post Cover image"
+            imgSrc={post.image}
+            fallbackSrc="https://res.cloudinary.com/daily-now/image/upload/f_auto/v1/placeholders/1"
```

Comment:
Random question, do we want to keep this with the lazy loading or move to the native one?

Reply:
Ah, yes. It is our mission to move everything to the native one, let's refactor this as well. Thank you!